distributor-api-v1/net-pricing.md

```diff
@@ -0,0 +1,143 @@
+# Pricing
+
+**Net price** is an amount that does not have taxes included.  
+**Gross price** is an amount that already includes a net price + taxes. Examples of this are below in the responses to API calls.
+
+In order to use net pricing in distributor, the "Amount" object, that is returned from the API, has to be used. In every response that has something to do with pricing, the API returns two objects:
```

Comment:
document only the state we are keeping, do not mention the costs, nor in examples.